server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JMAPModule.java

```diff
@@ -129,6 +130,7 @@ protected void configure() {
 
         Multibinder.newSetBinder(binder(), EventListener.GroupEventListener.class).addBinding().to(PropagateLookupRightListener.class);
         Multibinder.newSetBinder(binder(), EventListener.ReactiveGroupEventListener.class).addBinding().to(MailboxChangeListener.class);
+        Multibinder.newSetBinder(binder(), EventListener.ReactiveGroupEventListener.class).addBinding().to(PushListener.class);
```

Comment:
We need to register this on the JMAP event bus (not on the regular event bus)

Reply:
I registered it at `server/container/guice/protocols/jmap/src/main/java/org/apache/james/modules/protocols/JmapEventBusModule.java` 
```java
 Multibinder.newSetBinder(binder(), EventListener.ReactiveGroupEventListener.class).addBinding().to(PushListener.class);
 ```
But It still not working

Reply:
`server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/draft/JMAPModule.java` is the regular event bus?